package.json

```diff
@@ -77,6 +78,7 @@
     "@wikimedia/codex-design-tokens": "^0.13.0",
     "husky": "^4.2.3",
     "lint-staged": "^10.0.9",
+    "nodemon": "^3.0.1",
```

Comment:
@varun-s22 Why do we need nodemon as a dependency here? Is it required for Docker?

Reply:
> @varun-s22 Why do we need nodemon as a dependency here? Is it required for Docker?

hi anmol,
yeah i thought it would restart the service on crashing